code/modules/reagents/reagent_containers/inhaler.dm

```diff
@@ -0,0 +1,117 @@
+//Inhalers
+//Just like hypopsray code
+/obj/item/chems/inhaler
+	name = "autoinhaler"
+	desc = "A rapid and safe way to administer small amounts of drugs into the lungs by untrained or trained personnel."
+	icon = 'icons/obj/syringe.dmi'
+	item_state = "autoinjector"
+	icon_state = "autoinhaler"
+	center_of_mass = @"{'x':16,'y':11}"
+	unacidable = TRUE
+	amount_per_transfer_from_this = 5
+	volume = 5
+	w_class = ITEM_SIZE_SMALL
+	possible_transfer_amounts = null
+	atom_flags = ATOM_FLAG_OPEN_CONTAINER
+	slot_flags = SLOT_LOWER_BODY
+	origin_tech = "{'materials':2,'biotech':2}"
+	var/used = FALSE
+	matter = list(
+		/decl/material/solid/fiberglass = MATTER_AMOUNT_REINFORCEMENT,
+		/decl/material/solid/metal/aluminium = MATTER_AMOUNT_TRACE
+	)
+	var/list/starts_with = null /// A lazylist of starting reagents. Example: list(/decl/material/liquid/adrenaline = 5)
+	var/band_color = COLOR_GREEN
```

Comment:
Could this be generalized on `/chems`? Doesn't feel correct to have it copied on two very adjacent types.